corehq/apps/auditcare/utils/migration.py

```diff
@@ -22,30 +22,33 @@ def get_next_batch_start(self):
 
     def generate_batches(self, worker_count, batch_by):
         batches = []
-        # todo : Change get_fixed_start_date_for_sql to something generic
-        cutoff_time = get_fixed_start_date_for_sql()
-        if not cutoff_time:
-            cutoff_time = datetime.now()
         with cache.lock(self.start_lock_key, timeout=10):
             start_datetime = self.get_next_batch_start()
+            print(start_datetime)
```

Comment:
Not sure why I didn't catch this before, but print statements are not allowed except in management commands. I realize this utility function is only used by a management command, so it's a bit of a gray area. Is this even needed except for debugging? If yes, consider using `logging` instead.